ChangeLog

```diff
@@ -1,3 +1,40 @@
+2019-11-28  Dirk Eddelbuettel  <edd@debian.org>
+
+	* DESCRIPTION (Version, Date): Roll minor version
+
+	* .travis.yml: Load development version of RcppCCTZ from GitHub
+
+2019-11-27  Leonardo Silvestri  <lsilvestri@ztsdb.org>
```

Comment:
More current date, maybe?   

(Also, generally, ChangeLog is per "chunk" of changes, similar to git logs, maybe a a little more coarse.  Here, it is of course a collection of a LOT of changes so may as well say so.  Your code, your reference -- whatever feels rights.)

Reply:
I changed the date part, and added two header lines, see 18bc559